FreeRTOS/FreeRTOS
# Add FreeRTOS_Plus_WolfSSL_FIPS_Ready_Windows_Simulator

Add FreeRTOS_Plus_WolfSSL_FIPS_Ready_Windows_Simulator

Description
-----------
This demo project is created to show how the wolfSSL-FIPS-Ready version works and how to use it to build with the demo application.

This demo project contains the latest wolfSSL-FIPS-Ready version (v4.5.0). Also, in this demo, the unused wolfSSL files for building the demo application have been removed.

Test Steps
-----------
Build RTOSDemo.exe with the solution file in the FreeRTOS-Plus/Demo/FreeRTOS_Plus_WolfSSLFIPS_Ready_Windows_Simulator and then run it with Debugger.

However, the demo application will fail for the first run. An error is expected at this point. Extra steps are required to build the application including wolfSSL-FIPS-Ready version.  Please refer the "How to build and run the Demo application" section in the FreeRTOS-Plus/Demo/FreeRTOS_Plus_WolfSSL_FIPS_Ready_Windows_Simulator/README_wolfSSL_FIPS_ready.md. 

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
